ipamutils/utils.go

```diff
@@ -2,6 +2,8 @@
 package ipamutils
 
 import (
+	"fmt"
+	"github.com/sirupsen/logrus"
```

Comment:
all the building libraries have to be on top and followed by blank line and user defined imports. 
```
import(
     "fmt"
     "net"
    "sync"

    "github.com/sirupsen/logrus"
)

Reply:
Done